lib/plugins/aws/package/compile/events/apiGateway/lib/method/integration.js

```diff
@@ -129,15 +129,23 @@ module.exports = {
       const integrationResponseHeaders = [];
 
       if (http.cors) {
-        // TODO remove once "origins" config is deprecated
         let origin = http.cors.origin;
+
+        // TODO remove once "origins" config is deprecated
         if (http.cors.origins && http.cors.origins.length) {
           origin = http.cors.origins.join(',');
         }
 
         _.merge(integrationResponseHeaders, {
           'Access-Control-Allow-Origin': `'${origin}'`,
         });
+
+        // Only set Access-Control-Allow-Credentials when explicitly allowed (omit if false)
+        if (http.cors.allowCredentials) {
+          _.merge(integrationResponseHeaders, {
+            'Access-Control-Allow-Credentials': `'${http.cors.allowCredentials}'`,
+          });
```

Comment:
Let's use plain JavaScript, and literally assign `"true"`, as it's a boolean value

```javascript
integrationResponseHeaders['Access-Control-Allow-Credentials'] = "true"
```